uber/site_sections/emails.py

```diff
@@ -1,7 +1,7 @@
 from uber.common import *
 
 
-@all_renderable(c.PEOPLE)
+@all_renderable(c.PEOPLE, c.REG_AT_CON)
```

Comment:
-1 on this - see https://github.com/magfest/ubersystem/issues/2056 on what I think we should do instead.  This would let reg at con folks do things like click "Approve" on pending emails.  (Not that I think people would actually do that, but we shouldn't allow it.)